libs/design-system/web-component-card/src/lib/web-component-card.component.scss

```diff
@@ -0,0 +1,59 @@
+:host {
+  display: block;
+  position: relative;
+  border-radius: 0.5rem;
+  overflow: hidden;
+  background-color: var(--sys-surface-container-low);
+  box-shadow: 0px 5px 16px 0px rgba(from var(--sys-shadow) r g b / 0.24);
```

Comment:
Use `rgb`. `rgba` is a legacy alias according to mdn (https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb#syntax)